ocs_ci/ocs/bucket_utils.py

```diff
@@ -1912,3 +1964,142 @@ def create_aws_bs_using_cli(
         f"--target-bucket {uls_name} --region {region}",
         use_yes=True,
     )
+
+
+def get_object_creation_time_from_noobaa_db(bucket_name, object_key):
+    """
```

Comment:
Thanks for pointing this out! Previously this would only return the string that represents the creation time of a single object, but having to define the output in the docstring allowed me to rethink and extend the utility of this function.

After my last commit this function now allows to return the creaton time of multiple objects by returning a dictionary, and instead of just returning the times as strings I've converted them to datetime objects.